lib/matplotlib/tests/test_pickle.py

```diff
@@ -236,7 +236,7 @@ def test_grid():
     fig = manager.canvas.figure
     ax = fig.add_subplot(1, 1, 1)
     ax.grid()
-    # Drawing the grid triggers instance methods to be attached
+    # Drawing the grid used to trigger instance methods to be attached
     # to the Line2D object (_lineFunc).
```

Comment:
Good point--I didn't notice that this was in a test.  Yes, I think removing the test makes perfect sense.